packages/cli/tests/unit/agentInstall/command.spec.ts

```diff
@@ -18,7 +18,6 @@ tmp.setGracefulCleanup();
 
 const invokeCommand = (
   projectDir: string,
-  projectType: string,
```

Comment:
We don't need this anymore because the installer will automatically identify the correct project type.